pkg/webhook/instance_admission.go

```diff
@@ -119,12 +126,13 @@ func handleUpdate(ia *InstanceAdmission, req admission.Request) admission.Respon
 		return admission.Errored(http.StatusInternalServerError, err)
 	}
 
-	// we explicitly ignore Metadata updates
-	if reflect.DeepEqual(old.Spec, new.Spec) && reflect.DeepEqual(old.Status, new.Status) {
-		return admission.Allowed("")
+	oldOv, err := old.GetOperatorVersion(ia.client)
```

Comment:
Technically, you only need to fetch the old `OV` if this is an upgrade, otherwise, they're the same. Given that the upgrades are rare, I think it's worth an additional `if` here.